prepare_podman_compose.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# make sure selinux is off
+sestatus | grep '^Current mode:.*enforcing$'
+if [ $? -eq 0 ]; then
+  echo "Selinux is enforcing.Set it to permissive mode before using podman-compose"
```

Comment:
Yeah, I don't think podman-compose handles the `:z` modifier on volumes properly.